src/lib/tests/fixture_app.cpp

```diff
@@ -12,7 +12,7 @@ SISL_OPTION_GROUP(
     (num_shards, "", "num_shards", "number of shards", ::cxxopts::value< uint64_t >()->default_value("20"), "number"),
     (num_blobs, "", "num_blobs", "number of blobs", ::cxxopts::value< uint64_t >()->default_value("50"), "number"));
 
-SISL_LOGGING_INIT(logging, HOMEOBJECT_LOG_MODS)
+SISL_LOGGING_DEF(logging, HOMEOBJECT_LOG_MODS)
```

Comment:
`_DEF` is meant to be in a library translation unit, otherwise SM will have to do this too. I suggest `homobject_impl.cpp` (likely the heap test too since it doesn't link homeobject i think?)

Reply:
ACK, makes sense. I will change this to a library.

Reply:
Just for clarity:
`..._DECL` : Just to make `LOGMOD...(mod,` compile, can appear any number of times
`..._DEF` : Provide symbol to linker (once per module)
`..._INIT` : Once per executable; loads modules into the global map to make them settable.